### ISSUE TITLE
configuration documentation contradicts itself

### ISSUE THREAD

**Issue by mbdevpl** [landscapeio/landscape-issues#237]
After looking closely at https://docs.landscape.io/configuration.html

Example file contains these lines:

```
doc-warnings: yes
test-warnings: no
```

While later documentation contains these:

```
doc-warnings <bool>
(...)
test-warnings <bool>
(...) The value can be true or false.
```

I understand that the config parser probably accounts for more than `true` and `false` so some unexpected values like `1`, `0`, `yes` and `no` are probably handled, but in my humble opinion the example in documentation should follow the recommendations strictly.

**Comment by carlio** [comment]
Ah true, thank you!  The basic thought is that the config file is just YAML so any YAML boolean value is valid (yes, no, true, false) so do you think it's better to be strict or to just explain that better?

**Comment by mbdevpl** [comment]
I find documentation easier to follow if it is consistent. So if it's `true`/`false` here it shouldn't be `Y`/`N` somewhere else. Even if there are actually more allowed values, I'd skip them and just mention that they exist in general to avoid confusion for new users (like me):

> The value can be true or false (other equivalent values allowed by YAML are also accepted).

On the other hand the configuration parsing engine should account for some variation, like it most certainly does (disclaimer: I didn't actually check) and follow rules of YAML, e.g.:

```
{ Y, true, Yes, ON  }    : Boolean true
{ n, FALSE, No, off }    : Boolean false
```

It's kind of similar with that in different cases the `key: value` follows different formatting. What I see after looking closely at https://docs.landscape.io/configuration.html and https://docs.landscape.io/advanced.html

sometimes it's simply

```
key: value
```

sometimes it's:

```
key:
  value
```

and sometimes it's:

```
key:
  - value
```

and indent differs! However, the underlying rules about this seem to not be described anywhere.

Assorted examples, not to be interpreted as single file:

```
python-targets:
    - 2
    - 3
pep8:
    full: true
pep8:
    none: true
pep8:
  disable:
    - E711
    - E712
pylint:
  run: false
pylint:
  options:
    docstring-min-length: 10
```

Basically even if parser allows various indentation style, all examples should follow some "best practice". E.g. I'm not sure which one is best:

```
pylint:
  disable:
    - eval-used
  options:
    docstring-min-length: 10
```

or:

```
pylint:
  disable:
    - eval-used
  options:
    - docstring-min-length: 10
```

or:

```
pylint:
  - disable:
    - eval-used
  - options:
    - docstring-min-length: 10
```

**Comment by mbdevpl** [comment]
BTW, Travis's docs at https://docs.travis-ci.com/user/customizing-the-build/ also have some of these issues :) I wonder why I didn't notice that earlier... Maybe because the commands like `build` or `install` seem more familiar to me than options used by Landscape, so it's easier to follow even without perfect clarity.

Anyways, it's not that big of a deal, just would be nice if those nuances could be made 100% clear. If those docs are in some public repo (probably not) I'd make a PR myself.

**Comment by carlio** [comment]
@mbdevpl Yep they are: )  here: https://github.com/landscapeio/landscape-docs

**Comment by mbdevpl** [comment]
Ah, sorry, I didn't search well enough. Then, I'll clone those...

**Comment by mbdevpl** [comment]
@carlio I've submitted my changes, please let me know whether you think they are really an improvement :)

**Comment by mbdevpl** [comment]
@carlio Since https://github.com/landscapeio/landscape-docs/pull/5 was merged, I'm closing this. Thanks for very rapid response.
